datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/types/auth/mappers/AccessTokenMetadataMapper.java

```diff
@@ -0,0 +1,47 @@
+package com.linkedin.datahub.graphql.types.auth.mappers;
+
+import com.linkedin.access.token.DataHubAccessTokenInfo;
+import com.linkedin.data.DataMap;
+import com.linkedin.datahub.graphql.generated.EntityType;
+import com.linkedin.datahub.graphql.generated.AccessTokenMetadata;
+import com.linkedin.datahub.graphql.types.common.mappers.util.MappingHelper;
+import com.linkedin.datahub.graphql.types.mappers.ModelMapper;
+import com.linkedin.entity.EntityResponse;
+import com.linkedin.entity.EnvelopedAspectMap;
+import com.linkedin.metadata.Constants;
+import javax.annotation.Nonnull;
+
+
+public class AccessTokenMetadataMapper implements ModelMapper<EntityResponse, AccessTokenMetadata> {
+
+  public static final AccessTokenMetadataMapper INSTANCE = new AccessTokenMetadataMapper();
+
+  public static AccessTokenMetadata map(@Nonnull final EntityResponse entityResponse) {
+    return INSTANCE.apply(entityResponse);
+  }
+
+  @Override
+  public AccessTokenMetadata apply(final EntityResponse input) {
```

Comment:
Is the input guaranteed to be non-null? 

If yes, consider marking these type of things @Nonnull so IDE can help us out